USAGE.md

```diff
@@ -843,3 +848,28 @@ EOF
 > When not enabling globbing for the `bound_claims_type`, the authentication on the feature branches will not work properly and the GitHub action will fail, because the source ref or `job_workflow_ref` ends in `@refs/heads/<NAME_OF_FEATURE_BRANCH>`:
 > * https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#understanding-the-oidc-token
 > * https://github.com/hashicorp/vault-action#jwt-with-github-oidc-tokens
+
+## How to view Metrics in Kibana
+
+Vault can send telemetry data to a [Statsd compatible listener](https://developer.hashicorp.com/vault/docs/configuration/telemetry), for instance, a Metricbeat.
+
+The Metricbeat in trun can send the telemetry data to an Elasticsearch instance.
+
+The metric data from Elasticsearch can be visualized in a Kibana dashboard.
+
+> :info: Alternative to Filebeat and MetricBeat exists, the "Elastic Agents":
+> https://docs.elastic.co/en/integrations/hashicorp_vault
+
+Export/Import: Go to "Stack Management > Saved Objects" to export/import the dashboard (https://www.elastic.co/guide/en/kibana/current/managing-saved-objects.html)
+
+Todo: Add and test API call here to import the `kibana-dashboard.ndjson` file automatically
+
+https://www.elastic.co/guide/en/kibana/current/saved-objects-api-import.html
+
+```bash
+curl -XPOST "http://127.0.0.1:5601/api/saved_objects/_import -H "kbn-xsrf: true" -d @kibana-dashboard.ndjson"
```

Comment:
This needs to be tested. We set `disable_hostname=true` in the listener to make the dashboard easy copy/pastable. Otherwise, the metrics have the domain name included which is different on each system.